wp-content/themes/cpipr/page-donaciones.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Single Post Template: One Column (Standard Layout)
+ * Template Name: One Column (Standard Layout)
```

Comment:
This header should be changed to reflect that it's a template for a specific page.

It looks like this template isn't showing up on staging for logged-out users: http://staging.agile-engine.flywheelsites.com/donaciones/